test/auth/oauth-with-idtoken-validation.tests.js

```diff
@@ -162,6 +166,10 @@ describe('OAUthWithIDTokenValidation', function() {
           return new Promise(res => res({ id_token: 'foobar' }));
         }
       };
+      OAUthWithIDTokenValidationProxy = proxyquire('../../src/auth/OAUthWithIDTokenValidation', {
```

Comment:
nit, fix caps. You'd need to change usages as well

```suggestion
      OAuthWithIDTokenValidationProxy = proxyquire('../../src/auth/OAUthWithIDTokenValidation', {
```